pages/common/less.md

```diff
@@ -33,3 +29,7 @@
 - Exit:
 
 `q`
+
+- Read pdf files with less
+
+`lesspipe {{filename}}.pdf | less`
```

Comment:
Generally, we try to avoid documenting other commands on a tldr page. I'm not sure this example is really appropriate here, but have you considered writing a new page for `lesspipe`? A quick check reveals that we don't yet have a page for it - and I think this example would be much better served on a dedicated page :-)

Reply:
lesspipe is not meant to be a stand alone tool, its always used with less; also this example expresses how less can read pdf files via assistance from lesspipe. We can also create a separate page but I am pretty sure this example belongs here.

Ask yourself this where does petrol belong? A petrol pump or in your car's tank? Answer: Both!

Reply:
Ah, I see. Thanks for the clarification! Is `lesspipe` installed when `less` is installed? Or is it another package that needs installing separately? I seem to have it installed already on my laptop here without doing anything, so I'm inclined to believe it's the former.

Anyway, to fix the linter issue you need a colon there at the end of the description text :-)

Reply:
It depends on the package you got less from, usually lesspipe comes with less.

Reply:
Tried on Ubuntu, lesspipe is here, but not in macOS.